### PR TITLE
Network watcher TA and flow logs

### PR DESCRIPTION
**Pull request by khushal08**
Recommendations for network watcher traffic analytics and NSG/VNET flow logs

NA

### Breaking Changes

NA

## As part of this pull request I have

- [x] Read the [Contribution Guide](https://azure.github.io/Azure-Proactive-Resiliency-Library-v2/contributing) and ensured this PR is compliant with the guide
- [x] Checked for duplicate [Pull Requests](https://github.com/Azure/Azure-Proactive-Resiliency-Library-v2/pulls)
- [x] Associated it with relevant [GitHub Issues](https://github.com/Azure/Azure-Proactive-Resiliency-Library-v2/issues) or ADO Work Items (Internal Only)
- [x] Ensured my code/branch is up-to-date with the latest changes in the `main` [branch](https://github.com/Azure/Azure-Proactive-Resiliency-Library-v2/tree/main)
- [x] Ensured PR tests are passing
- [x] Performed testing and provided evidence (e.g. screenshot of output) for any changes associated to ARG queries
- [x] Updated relevant and associated documentation (e.g. Contribution Guide, Docs etc.)

### CODE REVIEW

File: azure-resources/Network/networkWatchers/recommendations.yaml [hunk 1]
```diff
@@ -54,3 +54,41 @@
...
+  recommendationImpact: High
+  recommendationResourceType: Microsoft.Network/networkWatchers
+  recommendationMetadataState: Active
+  longDescription: |
+    Improves monitoring and security for Azure and Hybrid connectivity
+  potentialBenefits: Improves monitoring and security for Azure connectivity
+  pgVerified: true
+  publishedToLearn: false
+  publishedToAdvisor: false
+  automationAvailable: arg
```
Comment (ejhenry) [inline azure-resources/Network/networkWatchers/recommendations.yaml:71]:
```suggestion
  automationAvailable: no
```

File: azure-resources/Network/networkWatchers/recommendations.yaml [hunk 2]
```diff
@@ -54,3 +54,41 @@
...
+  recommendationImpact: High
+  recommendationResourceType: Microsoft.Network/networkWatchers
+  recommendationMetadataState: Active
+  longDescription: |
+    Improves monitoring, security and troubleshooting for Azure and Hybrid connectivity
+  potentialBenefits: Improves monitoring, security and troubleshooting.
+  pgVerified: true
+  publishedToLearn: false
+  publishedToAdvisor: false
+  automationAvailable: arg
```
Comment (ejhenry) [inline azure-resources/Network/networkWatchers/recommendations.yaml:90]:
```suggestion
  automationAvailable: no
```

**Comment by ejhenry** [comment]
@Azure/aprl-networking please review

**Review by jimays-avila — APPROVED** [review]

**Review by oZakari — APPROVED** [review]
LGTM

**Pull request merged**